Doc/reference/datamodel.rst

```diff
@@ -1145,7 +1145,7 @@ Basic customization
    class constructor expression.  If a base class has an :meth:`__init__`
    method, the derived class's :meth:`__init__` method, if any, must explicitly
    call it to ensure proper initialization of the base class part of the
-   instance; for example: ``BaseClass.__init__(self, [args...])``.
+   instance; for example: ``super().__init__(self, [args...])``.
```

Comment:
Thanks for explaining!  That's why I was reading this section in the first -- to understand super(), __new__, and __init__.   :-)    Your explanation makes it clearer.  I've made this change.